README.md

```diff
@@ -92,7 +92,7 @@ rav1e has several optional features that can be enabled by passing --features to
 
 ### Online Analyzer
 
-If your `.ivf` file is hosted somewhere (and CORS is enabled on your web server) you can use:
+If your `.ivf` file is hosted somewhere (and CORS is enabled on your webserver) you can use:
```

Comment:
This may be more personal preference, but as an example the [Wikipedia article](https://en.wikipedia.org/wiki/Web_server) includes the space and it seems the more common usage is to spell it as two words.